flavors/python-3.8-minimal-EXASOL-6.2.0/flavor_base/udfclient_deps/packages/apt_get_packages

```diff
@@ -1,6 +1,6 @@
 coreutils|8.30-3ubuntu2
-locales|2.31-0ubuntu9.7
+locales|2.31-0ubuntu9.9
 libnss-db|2.2.3pre1-6build6
 libzmq3-dev|4.3.2-2ubuntu1
 libprotobuf-dev|3.6.1.3-2ubuntu5
-libssl-dev|1.1.1f-1ubuntu2.13
+libssl-dev|1.1.1f-1ubuntu2.15
```

Comment:
This looks affected since CVE states that the fix was introduced in 1.1.1p (> 1.1.1f, I assume).

Reply:
[Here](http://changelogs.ubuntu.com/changelogs/pool/main/o/openssl/openssl_1.1.1f-1ubuntu2.15/changelog) is the changelog for this package.
Please note that Canonical maintains their own versions, and backports critical issues by their own.

Reply:
Awesome! In this case the PR is approved.